content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -67,6 +67,12 @@ Endpoint Slices can act as the source of truth for kube-proxy when it comes to
 how to route internal traffic. When enabled, they should provide a performance
 improvement for services with large numbers of endpoints.
 
+## Address Types
+
+The default EndpointSlice address type is IP. This can include both IPv4 and
```

Comment:
```suggestion
The default EndpointSlice address type is `IP`. This can include both IPv4 and
```
because (I think) that makes it clear it's a value.

Reply:
Hey @sftim, thanks for the feedback! We actually ended up changing this to not have a default value, I've updated the docs to reflect that.